cypress-tests/cypress/e2e/spec/Payment/00029-IncrementalAuth.cy.js

```diff
@@ -88,6 +88,7 @@ describe("[Payment] Incremental Auth", () => {
       const newData = {
         ...data,
         Request: { amount_to_capture: data.Request.amount_to_capture + 2000 },
+        Response: getConnectorDetails(data, "ResponseCustom"),
```

Comment:
This would not work, even if you log `getConnectorDetails(data, "ResponseCustom")` in the console, it would log 

<img width="200" src="https://github.com/user-attachments/assets/b1aea6de-b77e-4f85-9d3b-884f6c5a52ec" />

```suggestion
        Response: data.ResponseCustom,
```

Use this instead.